frontend/src/theme/utils.ts

```diff
@@ -694,9 +694,19 @@ export function getSequentialColorsArray(theme: Theme): string[] {
 }
 
 export function getDivergingColorsArray(theme: Theme): string[] {
-  return hasLightBackgroundColor(theme)
-    ? getBlueArrayDesc(theme)
-    : getBlueArrayAsc(theme)
+  const { colors } = theme
```

Comment:
nit: it is expected that we are using the same colors for dark & light themes in this case, right?

Reply:
<img width="574" alt="Screen Shot 2022-11-16 at 9 19 54 AM" src="https://user-images.githubusercontent.com/16749069/202249381-4cb2aa18-9bd8-4a95-a4c2-c11f1b77d3e1.png">
Based on this figma doc, it does seem to be the case since the second bar is the same colors but with different opacity. Also, I don't think the diverging colors are really ever used- at least I couldn't reproduce it - so I think if this is a big deal, we can do a quick swap.